analysed-packages/ppp/version-2.4.9/ppp-2.4.9-SPDX2TV.spdx

```diff
@@ -0,0 +1,5119 @@
+FileChecksum: SHA1: 6e97fa5d26c8411bbd6e838695681f038b497dd2
+FileChecksum: SHA256: adee3ce0e7ba4e237820bbdf8bc0f1607c192c4330d7657266527543063c369d
+FileChecksum: MD5: 34613d21711e77b453f5070e4023c8c0
+LicenseConcluded: LicenseRef-GPL-1.0-or-later
+LicenseComments: <text> The information in the file is:
+" This manual is published under the GPL. All guidelines specified in the GPL apply here."
+Since no version is given, we concluded GPL-1.0-or-later as license for this file. </text>
+LicenseInfoInFile: LicenseRef-GPL-1.0-or-later
+LicenseInfoInFile: LicenseRef-GPL
+FileCopyrightText: NOASSERTION
+
+##File
+
+FileName: ppp-2.4.9.tar.gz/ppp-2.4.9.tar/ppp-2.4.9/scripts/poff
+SPDXID: SPDXRef-item16155629
+FileChecksum: SHA1: 579893597678f81009db187f1214cb69f3b34588
+FileChecksum: SHA256: c77739a8777e8f6485e0db9e9153c208aeaf073246ab4aba3525380e6cf21530
+FileChecksum: MD5: b312a7490ad2cd818fe13198ea32edd7
+LicenseConcluded: LicenseRef-GPL-1.0-or-later
+LicenseComments: <text> The information in the file is:
+"Distributed under GNU GPL."
+Since no version is given, we concluded GPL-1.0-or-later as license for this file. </text>
+LicenseInfoInFile: LicenseRef-GPL
+FileCopyrightText: NOASSERTION
+
+##File
+
+FileName: ppp-2.4.9.tar.gz/ppp-2.4.9.tar/ppp-2.4.9/pppstats/pppstats.c
+SPDXID: SPDXRef-item16155688
+FileChecksum: SHA1: 9321a94b6217e19c878d2cd47fdb3a4ef9489ce3
+FileChecksum: SHA256: bf1b18ae8d1ecf00ccbec65da699c7d09bbd756786c7936833afafb6d7526fec
+FileChecksum: MD5: a790b96d3193fba005cde6c8265b1e50
+LicenseConcluded: LicenseRef-BSD-4.3TAHOE-28c7562c36393369030e4e3b3f7fc235
+LicenseInfoInFile: LicenseRef-BSD
+LicenseInfoInFile: LicenseRef-scancode-bsla
+LicenseInfoInFile: LicenseRef-BSD-4.3TAHOE
+FileCopyrightText: <text> Copyright (c) 1989 Regents of the University of California. All rights reserved. </text>
+FileAttributionText: <text> This software was developed by the University of California, Berkeley. </text>
+
+##File
+
+FileName: ppp-2.4.9.tar.gz/ppp-2.4.9.tar/ppp-2.4.9/pppd/md5.h
+SPDXID: SPDXRef-item16155491
+FileChecksum: SHA1: 7c008601faa7d5e5af27eea011a1439c990a16f9
+FileChecksum: SHA256: abd3c39b23119924bdd1b29f9e95959868a67f652a11f691ee5033b9c31c5425
+FileChecksum: MD5: b83cfbd4161a239539200a84ff062385
+LicenseConcluded: LicenseRef-RSA-MD-f351336692a3613a12b45b03db5c347f
+LicenseInfoInFile: LicenseRef-RSA-MD
+FileCopyrightText: <text> Copyright (C) 1990, RSA Data Security, Inc. All rights reserved. </text>
+
+##File
+
+FileName: ppp-2.4.9.tar.gz/ppp-2.4.9.tar/ppp-2.4.9/pppd/md5.c
+SPDXID: SPDXRef-item16155499
+FileChecksum: SHA1: f592b31aa79e2290a73163b93723767bb1cb6908
+FileChecksum: SHA256: 53e42a0c520f219461542c28de004cdad4b91f961a836110ee00bd4a97aedd42
+FileChecksum: MD5: b34bd3dd67943d604903ca2844a1c15e
+LicenseConcluded: LicenseRef-RSA-MD-f351336692a3613a12b45b03db5c347f
+LicenseInfoInFile: LicenseRef-RSA-MD
+FileCopyrightText: <text> Copyright (C) 1990, RSA Data Security, Inc. All rights reserved. </text>
```

Comment:
Please add the following Attribution/Acknowledgment: "This software includes software derived from the RSA Data Security, Inc. MD5 Message-Digest Algorithm"